Include/internal/pycore_call.h

```diff
@@ -186,7 +186,7 @@ _PyObject_CallNoArgs(PyObject *func) {
 }
 
 
-extern PyObject *const *
+PyAPI_FUNC(PyObject *const *)
```

Comment:
Please add a comment to explain that this function is exposed for Cinder. Something like:

```
// Export for the Cinder project (gh-131776)
```

Same comment for all PyAPI_FUNC() additions.

Reply:
I didn't specifically mention Cinder, but I added the same comment elsewhere that it exists for re-generated interpreters.